src/main/java/net/apnic/whowas/rdap/config/RDAPConfiguration.java

```diff
@@ -23,6 +24,16 @@
 @ConfigurationProperties(prefix="rdap")
 public class RDAPConfiguration
 {
+    private static final Notice TRUNCATED_NOTICE;
+
+    static
+    {
+        TRUNCATED_NOTICE = new Notice(
+            "Search Policy",
+            "result set truncated due to limits",
```

Comment:
This should be one of the values from the registry.  "result set truncated due to unexplainable reasons" is probably the most appropriate.

Reply:
Done & Done